### ISSUE TITLE
generateSalt does not work

### ISSUE THREAD

**Issue by RobbertH** [sroddy/flutter_string_encryption#5]
When executing the following two lines of code:

```
final PlatformStringCryptor cryptor = new PlatformStringCryptor();
var dummy = await cryptor.generateSalt();
```

I get the following error

```
E/flutter (26486): [ERROR:topaz/lib/tonic/logging/dart_error.cc(16)] Unhandled exception:
E/flutter (26486): type 'Future<dynamic>' is not a subtype of type 'Future<String>' where
E/flutter (26486):   Future is from dart:async
E/flutter (26486):   Future is from dart:async
E/flutter (26486):   String is from dart:core
E/flutter (26486): 
E/flutter (26486): #0      PlatformStringCryptor.generateSalt (package:flutter_string_encryption/flutter_string_encryption.dart:71:45)
[...]
```

**Comment by sroddy** [comment]
Hi, sorry I was away and came back recently.
I'm going to fix the package and release a fixed version today

**Comment by sroddy** [comment]
It should be now fixed in version 0.0.2.

Closing, please reopen if that's not the case.

Thanks

**Comment by RobbertH** [comment]
Yep, after getting the new version and running `flutter clean`, everything works :grinning: 
Thanks!
